internal/iam/command_ipfilter_list.go

```diff
@@ -66,14 +67,17 @@ func (c *ipFilterCommand) list(cmd *cobra.Command, _ []string) error {
 	}
 	list := output.NewList(cmd)
 	for _, filter := range ipFilters {
-		list.Add(&ipFilterOut{
-			ID:              filter.GetId(),
-			Name:            filter.GetFilterName(),
-			ResourceGroup:   filter.GetResourceGroup(),
-			IpGroups:        convertIpGroupObjectsToIpGroupIds(filter),
-			OperationGroups: filter.GetOperationGroups(),
-			ResourceScope:   filter.GetResourceScope(),
-		})
+		filterOut := ipFilterOut{
+			ID:            filter.GetId(),
+			Name:          filter.GetFilterName(),
+			ResourceGroup: filter.GetResourceGroup(),
+			IpGroups:      convertIpGroupObjectsToIpGroupIds(filter),
+		}
+		if isSrEnabled {
+			filterOut.ResourceScope = filter.GetResourceScope()
+			filterOut.OperationGroups = filter.GetOperationGroups()
+		}
+		list.Add(filterOut)
```

Comment:
this needs to be list.Add(&filterOut), otherwise this breaks the whole list command!